packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -13,17 +13,32 @@ const { identifiers } = utils;
  * Therefore we have to keep an additional set of helpers/extensions to the database naming methods
  */
 
-export const COMPONENT_JOIN_TABLE_SUFFIX = 'components';
-export const DZ_JOIN_TABLE_SUFFIX = 'components';
-export const COMPONENT_INVERSE_COLUMN_NAME = 'component';
-export const COMPONENT_TYPE_COLUMN = 'component_type';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const COMPONENT_JOIN_TABLE_SUFFIX = (options: MetadataOptions) => 'components';
```

Comment:
If we're not using the options, why are we receiving them here? What prevents us from removing the options + the eslint disable rule?

Reply:
Same for the other ones below. Although if I remember correctly, those constants functions will be replaced by new ones in the other identifiers PR, right?

Reply:
Yeah, originally this was going to be needed to determine the short versions. It will be removed once the other PR is merged into this one, so for now I'll leave this to avoid adding merge conflicts for something that will be replaced.

Reply:
In fact, I'll go ahead and merge that in to this one now. I think it's been separate long enough to be reviewed but now it's getting confusing.